### PR TITLE
Fix upgradeability test

### LINKED ISSUE #2645: Unskip upgradeability test post DAO #1 upgrade.
Skipped in PR #2644

### PR DESCRIPTION
**Pull request by vzotova**
**Type of PR:**
- [X] Bugfix

**Required reviews:** 
- [X] 1

**What this does:**
Fix upgradeability test

**Issues fixed/closed:**
Fixes #2645

### CODE REVIEW

File: tests/contracts/test_contracts_upgradeability.py [hunk 1]
```diff
@@ -110,10 +110,13 @@ def deploy_base_contract(blockchain_interface: BlockchainDeployerInterface,
...
     latest_version, _data = blockchain_interface.find_raw_contract_data(contract_name, "latest")
-    try:
-        overrides = CONSTRUCTOR_OVERRIDES[contract_name][latest_version]
-    except KeyError:
-        overrides = dict()
+    raw_contracts = blockchain_interface._raw_contract_cache
+    overrides = dict()
+    if len(raw_contracts[contract_name]) != 1:
+        try:
+            overrides = CONSTRUCTOR_OVERRIDES[contract_name][latest_version]
```
Comment (KPrasch) [inline tests/contracts/test_contracts_upgradeability.py:117]: This is fine, here's a nitpick: 
```
overrides = CONSTRUCTOR_OVERRIDES[contract_name].get(latest_version, dict())
```

File: tests/contracts/test_contracts_upgradeability.py [hunk 2]
```diff
@@ -166,7 +169,7 @@ def test_upgradeability(temp_dir_path):
         contract_name = PolicyManagerDeployer.contract_name
         skip_policy_manager_test = skip_test(blockchain_interface, contract_name)
 
-        if not skip_adjudicator_test and not skip_staking_escrow_test and not skip_policy_manager_test:
+        if skip_adjudicator_test and skip_staking_escrow_test and skip_policy_manager_test:
```
Comment (derekpierre) [inline tests/contracts/test_contracts_upgradeability.py:172]: 😮

**Review by derekpierre — APPROVED** [review]
🎸 .

@KPrasch I think the circleci `cli` failure is due to the probationary period ending.

File: tests/contracts/test_contracts_upgradeability.py [hunk 3]
```diff
@@ -135,7 +137,6 @@ def skip_test(blockchain_interface: BlockchainDeployerInterface, contract_name:
     return force_skip or len(raw_contracts[contract_name]) == 1
 
 
-@pytest.mark.skip()  # TODO: Unskip this test
```
Comment (KPrasch) [inline tests/contracts/test_contracts_upgradeability.py:138]: 👍🏻

**Review by KPrasch — APPROVED** [review]

**Pull request merged**